wallet/src/actors/app/handlers/refresh_session.rs

```diff
@@ -0,0 +1,40 @@
+use actix::prelude::*;
+use serde::{Deserialize, Serialize};
+use witnet_futures_utils::ActorFutureExt2;
+
+use crate::{actors::app, model, types};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct RefreshSessionRequest {
+  pub session_id: types::SessionId,
+}
+
+#[derive(Serialize)]
+pub struct RefreshSessionResponse {
+  pub success: bool,
+}
+
+impl Message for RefreshSessionRequest {
+    type Result = Result<RefreshSessionResponse, app::Error>;
+}
+
+impl Handler<RefreshSessionRequest> for app::App {
+    type Result = <RefreshSessionRequest as Message>::Result;
+
+    fn handle(&mut self, msg: RefreshSessionRequest, ctx: &mut Self::Context) -> Self::Result {
+      let mut session = self.state.sessions.get_mut(&msg.session_id).unwrap();
```

Comment:
Avoid this `unwrap` by considering the case that the `get_mut` returns a `None`, in which case the `handle` could return a `false`. :)